content/blog/2021/04/2021-04-16-deprecated-ruby-runtime.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Deprecated ruby-runtime and associated plugins"
+tags:
+- ruby
+- plugins
+authors:
+- halkeye
+opengraph:
+  image: /images/post-images/2021-04-deprecating-ruby-runtime.png
```

Comment:
@MarkEWaite was the image editing expert :)

maybe one of https://emojipedia.org/broken-heart/ ?

Reply:
From the [Jenkins Social Media Covers](https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit#slide=id.gcde1b2bfe8_0_0) slides, here is an alternative:

![2021-04-deprecating-ruby-runtime-broken-heart png](https://user-images.githubusercontent.com/156685/115060220-db89d300-9ea4-11eb-9683-a94fd864e5c1.png)

Rename as appropriate

Reply:
I am not sure we want to risk giving the false impression that Jenkins no longer supports Ruby-based _projects_.

Reply:
Agree with Jesse, we still love Ruby (as much as we did before). The little used Ruby Runtime is just not something that can/should be used to write Jenkins plugins anymore.

Reply:
How about this as an alternate opengraph image in a positive sense rather than a negative sense:

![2021-04-deprecating-ruby-runtime](https://user-images.githubusercontent.com/156685/115084839-abebc280-9ec6-11eb-9054-d3d6aff67046.png)

Reply:
Those images have been discarded.  See the current proposal at https://github.com/jenkins-infra/jenkins.io/pull/4256#issuecomment-821823663